test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -106,6 +106,7 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
     }
 
     It "Should open the application that associates with extension '.txt'" -Skip:(!$isFullWin) {
+        "c:\windows\system32\notepad.exe" | Should Exist
```

Comment:
Ignore this for now, trying to figure out why the test succeeds on my machine but fails in AppVeyor